### PR TITLE
CORE-14296 - Suspended members are missing from membership package on member activation/re-activation

### PR DESCRIPTION
**Pull request by nikinagy**
We only send active member list to a member who gets activated/re-activated (DistributeMemberInfoActionHandler). We should send them the suspended information too, just like we do in the synchronisation services.
Member list usually gets sorted out by an immediate sync in the end, but if this sync does not happened fast enough, the e2e tests are failing.

### CODE REVIEW

**Review by yift-r3 — DISMISSED** [review]
LGTM

File: components/membership/membership-service-impl/src/test/kotlin/net/corda/membership/service/impl/actions/DistributeMemberInfoActionHandlerTest.kt [hunk 1]
```diff
@@ -204,7 +207,7 @@ class DistributeMemberInfoActionHandlerTest {
     }
     private val groupReader: MembershipGroupReader = mock {
         on { groupParameters } doReturn groupParameters
-        on { lookup() } doReturn allActiveMembers
+        //on { lookup() } doReturn allActiveMembers
```
Comment (williamvigorr3) [inline components/membership/membership-service-impl/src/test/kotlin/net/corda/membership/service/impl/actions/DistributeMemberInfoActionHandlerTest.kt]: Delete?

File: components/membership/membership-service-impl/src/test/kotlin/net/corda/membership/service/impl/actions/DistributeMemberInfoActionHandlerTest.kt [hunk 2]
```diff
@@ -123,6 +125,7 @@ class DistributeMemberInfoActionHandlerTest {
         mockMemberInfo(createHoldingIdentity("member-$it"))
     } + memberInfo + mgm
     private val activeMembersWithoutMgm = allActiveMembers - mgm
+    private val nonPendingMembersWithoutMgm = allActiveMembers + suspendedMemberInfo -mgm
```
Comment (williamvigorr3) [inline components/membership/membership-service-impl/src/test/kotlin/net/corda/membership/service/impl/actions/DistributeMemberInfoActionHandlerTest.kt]: Nit: `suspendedMemberInfo - mgm` (missing whitespace)

**Review by williamvigorr3 — COMMENTED** [review]
LGTM (I left two minor comments).

File: components/membership/membership-service-impl/src/test/kotlin/net/corda/membership/service/impl/actions/DistributeMemberInfoActionHandlerTest.kt (hunk 1 above)
Comment (nikinagy) [reply components/membership/membership-service-impl/src/test/kotlin/net/corda/membership/service/impl/actions/DistributeMemberInfoActionHandlerTest.kt]: addressed in [eccbb3e](https://github.com/corda/corda-runtime-os/pull/4011/commits/eccbb3e8247600b1f211cc7ac0f483fe042a72d3)

File: components/membership/membership-service-impl/src/test/kotlin/net/corda/membership/service/impl/actions/DistributeMemberInfoActionHandlerTest.kt (hunk 2 above)
Comment (nikinagy) [reply components/membership/membership-service-impl/src/test/kotlin/net/corda/membership/service/impl/actions/DistributeMemberInfoActionHandlerTest.kt]: addressed in [eccbb3e](https://github.com/corda/corda-runtime-os/pull/4011/commits/eccbb3e8247600b1f211cc7ac0f483fe042a72d3)

**Review by williamvigorr3 — APPROVED** [review]
LGTM

**Pull request merged**